integrations/access/servicenow/client.go

```diff
@@ -250,6 +260,24 @@ func (snc *Client) CheckHealth(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 	defer resp.RawResponse.Body.Close()
+
+	if snc.StatusSink != nil {
+		var code types.PluginStatusCode
+		switch {
+		case resp.StatusCode() == http.StatusUnauthorized:
+			code = types.PluginStatusCode_UNAUTHORIZED
+		case resp.StatusCode() >= 200 && resp.StatusCode() < 400:
+			code = types.PluginStatusCode_RUNNING
+		default:
+			code = types.PluginStatusCode_OTHER_ERROR
+		}
+		log := logger.Get(resp.Request.Context())
+		log.Infof("Emitting servicenow plugin status: %v", code)
+		if err := snc.StatusSink.Emit(ctx, &types.PluginStatusV1{Code: code}); err != nil {
+			log.WithError(err).Errorf("Error while emitting servicenow plugin status: %v", err)
+		}
```

Comment:
Sorry, I was not clear
My suggestion is to log the code only when it failed to `Emit`

Always logging seems a bit noisy (could be ok for debug level, tho).

So, something like this:
```suggestion
		if err := snc.StatusSink.Emit(ctx, &types.PluginStatusV1{Code: code}); err != nil {
			log := logger.Get(resp.Request.Context())
			log.WithError(err).WithField("code", resp.StatusCode()).Errorf("Error while emitting servicenow plugin status: %v", err)
		}
```